doc/syntax.md

```diff
@@ -94,38 +94,47 @@ type of the event has to be given in the `output` block.
 Currenly only spike output is supported.
 
 
-## Predefined functions, variables and keywords
+## Synaptic input
 
- `curr_sum` is a function that has two arguments. The first is a `shape` function (see Equations) or a function that is defined by an ODE plus initial values (see ODEs) $I$ of `t`. The second is a `spike` input buffer. Curr_sum takes every weight in the spike buffer and multiplies it with the `shape` function I_{\text{shape}} shifted by it's respective spike time $t_i$. I.e.
+NESTML has two dedicated functions to ease the summation of synaptic input.
 
-$\sum_{t_i\le t, i\in\mathbb{N}}\sum_{w\in\text{spikeweights}} w I_{\text{shape}}(t-t_i)=\sum_{t_i\le t, i\in\mathbb{N}} I_{\text{shape}}(t-t_i)\sum_{w\in\text{spikeweights}} w$.
- 
-When modeling post synaptic responses with delta functions, `curr_sum` is called with the keyword  `delta` as first argument instead of the `shape` function.
+`curr_sum` is a function that has two arguments. The first is a `shape` function (see [Equations](#equations)) or a function that is defined by an ODE plus initial values (see [Systems of ODEs](#systems-of-odes)) *I* of *t*. The second is a `spike` input buffer (see [Synaptic input](#synaptic-input)). `curr_sum` takes every weight in the `spike` buffer and multiplies it with the `shape` function *I*<sub>shape</sub> shifted by it's respective spike time *t<sub>i</sub>*. I.e.
```

Comment:
I suggest:
The first is a function *I*of *t* which is either a  `shape` function (see [Equations](#equations)) or a function that is defined by an ODE plus initial values (see [Systems of ODEs](#systems-of-odes)).